test/mockdniController.js

```diff
@@ -0,0 +1,3 @@
+angular.module('app.directives').controller('MockDniController',['$scope',function($scope){
+	$scope.fields = {dni: ""}
```

Comment:
It's just a matter of ease, because the functionality would remain the same, but you could use $scope.dni = '' and change it in the input too.